modules/client/examples/04-decoders/16-remove-multisig-addresses copy.ts

```diff
@@ -0,0 +1,30 @@
+/* MARKDOWN
+### Decode Remove Members Action (Multisig)
+*/
+import {
+  Context,
+  ContextPlugin,
+  MultisigClient,
+  MultisigPluginSettings,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+const multisigClient = new MultisigClient(contextPlugin);
+const data: Uint8Array = new Uint8Array([12, 56]);
+
+const settings: MultisigPluginSettings = multisigClient.decoding
+  .removeAddressesAction(data);
+
+console.log(settings);
+/*
+{
+  members: [
```

Comment:
I might call it `signers` or `addresses` as opposed to `members`, everywhere

Reply:
Again, following the contracts naming pattern, if we decide to change it i would do it in both places